lib/bottomnavigation/bottomnav.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math' as math;
+
+import 'package:SquanchyFlutter/eventdetails/eventdetails.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:SquanchyFlutter/favourites/favourites.dart';
+import 'package:SquanchyFlutter/schedule/schedule.dart';
+import 'package:SquanchyFlutter/venue/venuedetails.dart';
+
+class NavigationBar extends StatelessWidget {
```

Comment:
this seem like the main content which then contains also the navigation bar, worth extracting the nav bar to its own class and renaming this?

Reply:
Yeah good point, I made alot of this as I went!